prow/kube/client.go

```diff
@@ -37,6 +37,8 @@ import (
 )
 
 const (
+	TestContainerName = "test"
```

Comment:
nit: I feel like constants such as this should actually be shared, so that refactoring is just updating the single constant / potentially it's consumers instead of finding each re-definition.. /shrug

Reply:
This was a move of the constant, not a creation of another one -- couldn't use the original one due to import cycle

Reply:
thanks, this seems like the appropriate place -- I didn't look at the context for the podspec.go change thoroughly, my mistake.
/lgtm